src/Doctrine/DoctrineHelper.php

```diff
@@ -145,7 +145,7 @@ public function doesClassUsesAttributes(string $className): bool
 
     public function isDoctrineSupportingAttributes(): bool
     {
-        return $this->isDoctrineInstalled() && $this->attributeMappingSupport && $this->phpCompatUtil->canUseAttributes();
+        return $this->isDoctrineInstalled() && $this->attributeMappingSupport;
     }
 
```

Comment:
We will handle bringing doctrine related makers up to PHP8 standards in a separate PR to avoid too much bloat in this one.